docs/contents/dev-docs/release.md

```diff
@@ -146,10 +146,15 @@ Login succeeded
     something similar.
 
 Now publish all the controller images to the ECR public repository for
-controller images using the `scripts/helm-publish-charts.sh` script:
+controller image using the `scripts/helm-publish-charts.sh` script for each service:
+
+```bash
+for SERVICE in apigatewayv2 sns;\
```

Comment:
typically we place the `do` after the `;` and leave off the `\`, like so:

```bash
for SERVICE in apigatewayv2 sns; do
```

Reply:
Updated the document.